apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java

```diff
@@ -3356,7 +3350,16 @@ static void mapEobCommonGroupInpOutHHAHospiceSNF(
       Optional<Instant> lastUpdated,
       Optional<String> fiDocClmControlNum,
       Optional<LocalDate> fiClmProcDt,
-      C4BBInstutionalClaimSubtypes c4bbInstutionalClaimSubtype) {
+      C4BBInstutionalClaimSubtypes c4bbInstutionalClaimSubtype,
+      Optional<Character> claimQueryCode) {
```

Comment:
If we're saying claim query code is optional here, it should be optional where it's mapped in the other transformers (and have appropriate negative tests)